flatland_viz/src/spawn_model_tool.cpp

```diff
@@ -293,8 +291,10 @@ void SpawnModelTool::LoadPreview() {
   moving_model_node_->removeAllChildren();
   lines_list_.clear();
 
+  std::shared_ptr<rclcpp::Node> node = rclcpp::Node::make_shared("spawn_model_tool");  // TODO
```

Comment:
Not sure if it is a good way to create the node locally here or better as member variable?